core-bundle/src/Resources/contao/config/config.php

```diff
@@ -332,34 +333,35 @@
 // Back end form fields
 $GLOBALS['BE_FFL'] = array
 (
-	'text'           => TextField::class,
-	'password'       => Password::class,
-	'textStore'      => TextStore::class,
-	'textarea'       => TextArea::class,
-	'select'         => SelectMenu::class,
-	'checkbox'       => CheckBox::class,
-	'checkboxWizard' => CheckBoxWizard::class,
-	'radio'          => RadioButton::class,
-	'radioTable'     => RadioTable::class,
-	'inputUnit'      => InputUnit::class,
-	'trbl'           => TrblField::class,
-	'chmod'          => ChmodTable::class,
-	'picker'         => Picker::class,
-	'pageTree'       => PageTree::class,
-	'pageSelector'   => PageSelector::class,
-	'fileTree'       => FileTree::class,
-	'fileSelector'   => FileSelector::class,
-	'fileUpload'     => Upload::class,
-	'tableWizard'    => TableWizard::class,
-	'listWizard'     => ListWizard::class,
-	'optionWizard'   => OptionWizard::class,
-	'moduleWizard'   => ModuleWizard::class,
-	'keyValueWizard' => KeyValueWizard::class,
-	'imageSize'      => ImageSize::class,
-	'timePeriod'     => TimePeriod::class,
-	'metaWizard'     => MetaWizard::class,
-	'sectionWizard'  => SectionWizard::class,
-	'serpPreview'    => SerpPreview::class
+	'text'                    => TextField::class,
+	'password'                => Password::class,
+	'textStore'               => TextStore::class,
+	'textarea'                => TextArea::class,
+	'select'                  => SelectMenu::class,
+	'checkbox'                => CheckBox::class,
+	'checkboxWizard'          => CheckBoxWizard::class,
+	'radio'                   => RadioButton::class,
+	'radioTable'              => RadioTable::class,
+	'inputUnit'               => InputUnit::class,
+	'trbl'                    => TrblField::class,
+	'chmod'                   => ChmodTable::class,
+	'picker'                  => Picker::class,
+	'pageTree'                => PageTree::class,
+	'pageSelector'            => PageSelector::class,
+	'fileTree'                => FileTree::class,
+	'fileSelector'            => FileSelector::class,
+	'fileUpload'              => Upload::class,
+	'tableWizard'             => TableWizard::class,
+	'listWizard'              => ListWizard::class,
+	'optionWizard'            => OptionWizard::class,
+	'moduleWizard'            => ModuleWizard::class,
+	'keyValueWizard'          => KeyValueWizard::class,
+	'imageSize'               => ImageSize::class,
+	'timePeriod'              => TimePeriod::class,
+	'metaWizard'              => MetaWizard::class,
+	'sectionWizard'           => SectionWizard::class,
+	'serpPreview'             => SerpPreview::class,
+	'rootPageDependentModule' => RootPageDependentModule::class
```

Comment:
Why would you register a new fragment the old way?

Reply:
Possibly kind of "operational blindness" 😬

Reply:
Removed in f6b8d50ce.

Reply:
This is actually not the fragment, it's the backend form field type. Will readd.